netdata-installer.sh

```diff
@@ -735,6 +735,11 @@ if [ "${UID}" -eq 0 ]; then
         run chmod 4750 "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/xenstat.plugin"
     fi
 
+    if [ -f "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/perf.plugin" ]; then
+        run chown root:${NETDATA_GROUP} "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/perf.plugin"
```

Comment:
sorry for this nit picking, but you need to tabify these lines, as we use tabs on our bash scripts.

Check how the rest of the lines are and please fix xenstat,ioping and perf.plugin blocks